sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -62,21 +62,21 @@
 @pytest.mark.parametrize("loss", ("log_loss", "exponential"))
 def test_classification_toy(loss, global_random_seed):
     # Check classification on a toy dataset.
-    clf = GradientBoostingClassifier(
+    estimator = GradientBoostingClassifier(
```

Comment:
Two things here:

- `clf` for classifier is fine.
- we intend to modify code that is only related to the scope of the PR and not adding cosmetic change even if meaningful. We would reserve those in a different PR.

Thus, could you revert the `estimator` to `clf` for the test that we are not modifying in this PR.